test/test_sum.jl

```diff
@@ -0,0 +1,20 @@
+module TestSum
+
+using Test, BandedMatrices, Random
+
+r = brand(Float64,rand(1:10_000),rand(1:10_000),rand(-20:100),rand(-20:100))
```

Comment:
There's a chance this this generates a zero matrix -

```julia
julia> Random.seed!(27)
TaskLocalRNG()

julia> all(iszero, brand(Float64, rand(1:10_000), rand(1:10_000), rand(-20:100), rand(-20:100)))
true
```

Might be best to generate several random test cases incase the CI somehow only tests two zero matrices.

Reply:
Actually the best is to just seed the random number generator so its completely deterministic. Eg. `Random.seed!(0)`.

Reply:
Simulation shows that the probability of a zero matrix is only ~5% so, even if the RNG stream does change for newer Julia versions, it's probably unlikely that `Random.seed!(0)` gives zero so I agree.